### DISCUSSION TITLE
케릭터나 건물 쉐이더

### DISCUSSION THREAD

**Discussion by yoonjoohwan** [naverz/zepeto-studio-kor#774] [5. Q&A]
![image](https://user-images.githubusercontent.com/91707090/169194299-77c8ec9f-52d5-465f-9708-aa57ee84cb19.png)
유니티 상에서 케릭터의 쉐이더가 Wit/Eye 라고 되어있는데
저 쉐이더는 제페토에서 만들어서 쓰는것인가요?
빌드잇에서 사용하는 건물들의 쉐이더도 저 쉐이더 인가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 캐릭터에 사용된 Wit/Eye 쉐이더는 내부에서 제작한 기본 쉐이더입니다.
제페토 캐릭터는 기본적으로 아래 베이스 쉐이더로 구성되어 있습니다.
Eyelash : Wit/TransparentSoftEdgeWithColor
Eye : Wit/Eye
Body : Wit/Skin
Hair : Wit/Hair
기본 : Wit/Standard

그 외 아이템에 따라 커스텀 쉐이더를 사용하기도 합니다.
감사합니다.
